src/trait_manager/view/TraitView.ts

```diff
@@ -92,42 +112,27 @@ export default class TraitView extends Backbone.View {
     return this.model.get('value');
   }
 
-  setInputValue(value) {
-    const el = this.getInputElem();
-    el && (el.value = value);
-  }
-
   /**
    * On change callback
    * @private
    */
-  onValueChange(model, value, opts = {}) {
+  onValueChange(model: Trait, value: any, opts: any = {}) {
     if (opts.fromTarget) {
-      this.setInputValue(model.get('value'));
+      const el = this.getInputElem();
+      el && (el.value = value);
       this.postUpdate();
     } else {
-      const val = this.getValueForTarget();
-      model.setTargetValue(val, opts);
+      model.setTargetValue(value, opts);
     }
   }
 
   /**
    * Render label
    * @private
    */
-  renderLabel() {
-    const { $el, target } = this;
-    const label = this.getLabel();
-    let tpl = this.templateLabel(target);
-
-    if (this.createLabel) {
-      tpl =
-        this.createLabel({
-          label,
-          component: target,
-          trait: this,
-        }) || '';
-    }
+  private renderLabel() {
+    const { $el } = this;
+    let tpl = this.templateLabel;
```

Comment:
I know it doesn't make sense from the class perspective to have undefined methods but we can't simply remove methods publicly documented for customization like `createLabel`
https://grapesjs.com/docs/modules/Traits.html#update-layout